projects/prison-case-notes-to-probation/src/test/resources/application-test.yml

```diff
@@ -0,0 +1,17 @@
+spring:
+  datasource:
+    url: jdbc:h2:file:./projects/prison-case-notes-to-probation/test;MODE=Oracle;AUTO_SERVER=true;AUTO_SERVER_PORT=9092
```

Comment:
Should this file be in `src/integrationTest` rather than `src/test`?  I'm guessing we wouldn't need H2 or Hoverfly for the unit tests?

Reply:
Maybe better to keep it consistent so `application-integration-test.yml` - happy either way though.